std/variant.d

```diff
@@ -1459,8 +1451,7 @@ unittest
  *
  */
 
-// @@@ BUG IN COMPILER. THE 'STATIC' BELOW SHOULD NOT COMPILE
-static class VariantException : Exception
+class VariantException : Exception
```

Comment:
That line indicated that the compiler accepts invalid code - namely static classes at global scope are redundant.